taichi/ui/backends/vulkan/renderer.cpp

```diff
@@ -43,37 +44,40 @@ void Renderer::set_background_color(const glm::vec3 &color) {
 }
 
 void Renderer::set_image(const SetImageInfo &info) {
-  SetImage *s = get_renderable_of_type<SetImage>();
+  SetImage *s = get_renderable_of_type<SetImage>(VboAttribes::kAll);
```

Comment:
can probably use `kPosNormalUv` here. Although I don't think it would make any substantial difference.

Reply:
Huh, better not change the behavior here? I feel like other parts (especially `staging_buffer.py`) needs some tweak before this will work?